cmd/root.go

```diff
@@ -72,19 +75,22 @@ var RootCmd = &cobra.Command{
 	Use:               "chamber",
 	Short:             "CLI for storing secrets",
 	SilenceUsage:      true,
-	PersistentPreRun:  prerun,
+	PersistentPreRunE: prerun,
 	PersistentPostRun: postrun,
 }
 
 func init() {
 	RootCmd.PersistentFlags().IntVarP(&numRetries, "retries", "r", DefaultNumRetries, "For SSM or Secrets Manager, the number of retries we'll make before giving up; AKA $CHAMBER_RETRIES")
 	RootCmd.PersistentFlags().DurationVarP(&minThrottleDelay, "min-throttle-delay", "", 0, "DEPRECATED and no longer has any effect. Use retry-mode instead")
+	RootCmd.PersistentFlags().MarkDeprecated("min-throttle-delay", "use --retry-mode instead")
 	RootCmd.PersistentFlags().StringVarP(&retryMode, "retry-mode", "", store.DefaultRetryMode.String(),
 		`For SSM, the model used to retry requests
   `+aws.RetryModeStandard.String()+`
   `+aws.RetryModeAdaptive.String(),
 	)
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "", false, "Print more information to STDOUT")
+	RootCmd.PersistentFlags().MarkDeprecated("verbose", "use --log-level debug instead")
+	RootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "", "Log level")
```

Comment:
as a user of chamber, how do I know what the valid values for log-level is? Do we _need_ this level of granularity versus just a verbose flag?

Reply:
Removed `--log-level` and un-deprecated `--verbose`.